### DISCUSSION TITLE
airway-beacons-list-eastern-u-s/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#20] [Announcements]
# airway-beacons-list-eastern-u-s/

http://localhost:4000/airway-beacons-list-eastern-u-s/

**Comment by jengalas** [comment]
Posted by Charles Allen
February 1, 2023 at 8:10 pm
_Imported from WordPress_

BCN10 was in Hart County, Ga. (Hartwell) The Arrow is kept clean of leaves and grass by the property owners. BCN9 was also in Hart County, Ga. (Royston)

**Comment by jengalas** [comment]
Posted by Charles Allen
February 1, 2023 at 7:56 pm
_Imported from WordPress_

Thank You, Zhanna ! I live near the arrow in Hart County, Ga. (Hartwell) My Brother lives near the one in Madison County, Ga. (Ft Lamar) There is supposed to be another in Hart County, Ga. in the Royston area. We are looking for it.

**Comment by jengalas** [comment]
Posted by sam
December 4, 2022 at 2:56 pm
_Imported from WordPress_

Hello I have a rotating beacon , I would like to sell it not sure how much they are .

**Reply by jengalas** [reply]
Posted by Joseph
September 15, 2023 at 5:08 pm
_Imported from WordPress_

Did you sell it?

**Reply by jengalas** [reply]
Posted by Sherman
October 9, 2023 at 8:14 pm
_Imported from WordPress_

Please call me regarding your beacon
Sherman Swofford
864 706 9591
skyvisionsc@aol.com

**Comment by jengalas** [comment]
Posted by Gordon Tuthill
May 29, 2021 at 7:20 pm
_Imported from WordPress_

While surveying a property in Sharon Vermont I blundered onto the concrete base of a Beacon. the location is a little west of Vt 132 at the height of land. It is very close to the Sharon – Strafford town line.

**Comment by jengalas** [comment]
Posted by David Kachik
November 4, 2019 at 8:02 am
_Imported from WordPress_

I have KW2913 airway beacon tower 53 nittany mountain my dad won the bid for 15 dollars dismantled tower piece by piece around 1969 or 1970 put back up in clarence pa.for a cb radio antenna was the tower there before or after the the crash of chas ames?

**Reply by jengalas** [reply]
Posted by Sherman Swofford
October 9, 2023 at 8:16 pm
_Imported from WordPress_

Are you wanting to sell?
Please call
864 706 9591

**Comment by jengalas** [comment]
Posted by Zhanna
April 18, 2016 at 9:19 pm
_Imported from WordPress_

Jason writes:

**Does anyone have a wiring diagram for these lights? I recently acquired light #99 and was told it came from the Marana Air field in Arizona. I would like to restore the light to its original condition and functioning.**

Can anyone help him out? Reply and I’ll put you in touch.

**Reply by jengalas** [reply]
Posted by Joe miller
June 13, 2019 at 5:36 pm
_Imported from WordPress_

Does anyone have an idea where I could acquire a beacon? Thanks

**Reply by jengalas** [reply]
Posted by Rod Doell
March 8, 2021 at 3:41 pm
_Imported from WordPress_

Hi Joe, I have one that I am willing to part with.
Rod

**Reply by jengalas** [reply]
Posted by Garry Holcomb
February 27, 2022 at 1:41 pm
_Imported from WordPress_

Would love to have a beacon at my private airport

**Comment by jengalas** [comment]
Posted by Karin kaufman
March 10, 2016 at 12:44 pm
_Imported from WordPress_

sent a reply message about my propane lamp that has been burning for many year 800 ft. Mountain Above sea level near Westchester Airport as a possible thermal becan for hight near the airport Pon RI’d he . NY I do not see my letter or my became address yet. Concerned Karin

**Reply by jengalas** [reply]
Posted by Zhanna
March 10, 2016 at 2:10 pm
_Imported from WordPress_

Hi Karin,
I’ve approved both of your comments, but I’m not sure what you’re asking. It is possible the light is or has been used as an aid to navigation, but I would need more information to see if anyone knows more about it. Do you have coordinates of the location of the light?

**Comment by jengalas** [comment]
Posted by Karin kaufman
March 9, 2016 at 3:49 pm
_Imported from WordPress_

I have had a propane lamp lit fir many years on a mountain top near Westchester Aurport, it might be a helpful thermal signal for the hight of the mountain before the airport 800 ft above sea level. I thought it should be on charts as a night or day vision aid. Pound Ridge New York. Karin

**Comment by jengalas** [comment]
Posted by Zhanna
February 5, 2015 at 8:54 am
_Imported from WordPress_

I have received the following suggestion from Steve Bennett. I’m sure it will be helpful to many beacon-seekers! EarthExplorer has been added to the list of Resources.

**I have a suggestion for anyone hoping to find one. If you look in Google maps satellite view and see nothing but green trees, do not despair. You have other options. Use [http://earthexplorer.usgs.gov/](http://earthexplorer.usgs.gov/).**

**On the data sets tab expand aerial imagery and choose either “aerial single frames” or “high resolution orthoimagery”. The trick is look for imagery taken in the winter months when the leaves are down and you have a better view of the naked landscape without all the foliage blocking the view.**

**The aerial single frames images go farther back in time. It is not uncommon to find 1950s images there.**

**Comment by jengalas** [comment]
Posted by Zhanna
November 2, 2014 at 3:37 pm
_Imported from WordPress_

I have received the following message from Cameron. Please respond if you have any information that would help!

**Grandfather has an old airway beacon, he past away and don’t know anything about it, hope you could help me. It’s made by Sperry Gyroscope Co. INC.
Brooklyn N.Y. 110volts A.C. C.Y. PT-72834 SER-4017. He use to work at Maxwell Air Force Base back in the 50’s & 60’s and I think he got there.**

Images are available here:
[https://images.planetzhanna.com/airwaybeacons/artifacts/sperry-light-1.jpg](https://images.planetzhanna.com/airwaybeacons/artifacts/sperry-light-1.jpg)
[https://images.planetzhanna.com/airwaybeacons/artifacts/sperry-light-2.jpg](https://images.planetzhanna.com/airwaybeacons/artifacts/sperry-light-2.jpg)
[https://images.planetzhanna.com/airwaybeacons/artifacts/sperry-light-3.jpg](https://images.planetzhanna.com/airwaybeacons/artifacts/sperry-light-3.jpg)

**Comment by jengalas** [comment]
Posted by Zhanna
November 6, 2014 at 9:05 am
_Imported from WordPress_

I’ve received the following reply from Dave DuPree:

Hi Cameron,

Your question about your grandfather’s old rotating beacon was forwarded to me from Zhanna via Steve Owen in Grants, NM.

> From the photographs, I can tell you the following: Your beacon was manufactured between 1927 and about 1934. It is most likely from the 1928-1931 period.
>
> Although it was manufactured to the 1927 Department of Commerce airway beacon specifications, I think your beacon may have been built for use at an airport. During that period, the federal government did not provide beacons to civil airports, so only the largest airports could afford them. The data plate on your beacon appears to be the type used for a civil beacon, rather than one built under a DoC contract. One way to confirm whether your beacon was built as an airway beacon is to look at the back of the drum for a 4″circular medallion cast into the aluminum that reads “U.S. Dept. of Commerce Airways” (photos attached). A Sperry beacon built for the private market would have the Sperry logo on the back of the drum, or the back could be completely smooth. In any event, both types of 24″ beacon are nearly identical and both are equally historic. (In addition to being used at the early civil airports, a number of those early 24″ beacons were mounted on skyscrapers, bridges, etc. during the late 1920’s and 1930’s.)
>
> Your beacon is almost certainly not from a military base, as it would be configured differently (dual drums, different lens, etc.).
>
> The beacon is in remarkably original condition, including an original type main lens. The four glass panels along the top of the drum were called “zenith” lenses, and they are among the features that can be used to date the beacon. I couldn’t tell for sure from the photos, but it appears that you may have the original type of automatic lamp changer inside the drum (relatively few of those have survived). Most of the surviving 24″ beacons today were in use for many decades – first on the airways and then at airports – so periodic refurbishments, modifications and upgrades often resulted in the loss of original components. The drive system looks pretty rough – it may or may not be restorable – but most everything appears to be there.
>
> Overall, a very nice artifact from the Golden Age of Aviation. Thanks for sharing the photos. If you can, send pictures of the data plate and the interior of the drum – from those, I can provide additional information. I’m glad your grandfather saved it from the scrap heap! Let me know if you have any other questions.
>
> Dave DuPree
>Prescott, AZ

Accompanying photos:
[https://images.planetzhanna.com/airwaybeacons/artifacts/DoC-Airways-medallion-1.jpg](https://images.planetzhanna.com/airwaybeacons/artifacts/DoC-Airways-medallion-1.jpg)
[https://images.planetzhanna.com/airwaybeacons/artifacts/DoC-Airways-medallion-2.jpg](https://images.planetzhanna.com/airwaybeacons/artifacts/DoC-Airways-medallion-2.jpg)
[https://images.planetzhanna.com/airwaybeacons/artifacts/Denver-Municipal-(Stapleton)-1929.jpg](https://images.planetzhanna.com/airwaybeacons/artifacts/Denver-Municipal-(Stapleton)-1929.jpg)

_Copyright for the third photo is unknown. It shows a brand new 24″ beacon, almost identical to Cameron’s, atop the new terminal building at Denver Municipal Airport (Stapleton) in 1929._

**Comment by jengalas** [comment]
Posted by Joseph Miller
July 2, 2020 at 6:25 pm
_Imported from WordPress_

What became of this beacon? Still around?

**Comment by jengalas** [comment]
Posted by Sherman
October 9, 2023 at 8:10 pm
_Imported from WordPress_

Our pilots association in Spartanburg SC will soon begin refurbishing a beacon site.
We would love to talk to you about your light.
Sherman Swofford
864 706 9591

**Comment by jengalas** [comment]
Posted by Frank Roncone
June 9, 2014 at 1:35 pm
_Imported from WordPress_

There was one located at DuPage County Airport I witnessed the removal sometime during the 1970’s. Bill Donahue airport manager at the time stated that was the purpose. He was an old time aviator and knew many facts dealing with aviation.

**Comment by jengalas** [comment]
Posted by Adam
April 17, 2014 at 10:26 pm
_Imported from WordPress_

Are there any arrows around Northeast Indiana/Northwest Ohio/Southern Michigan that are visible?

**Comment by jengalas** [comment]
Posted by Patricia
January 13, 2014 at 7:47 pm
_Imported from WordPress_

I live in Tennessee and my husband and I own property on Monteagle Mountain. My father in law and other old timers have often spoke of the beacon light along the bluff. Monteagle in located between Chattanooga and Nashville. We live on the Grundy / Marion County lines. Do you have any idea how I might find exactly where this beacon was.
Thanks,
Patricia

**Reply by jengalas** [reply]
Posted by Zhanna
January 19, 2014 at 3:15 pm
_Imported from WordPress_

Hello Patricia,

It doesn’t appear to be in the NGS database, but there are many other beacons. If you take a look at the map here (United States airway maps. no. 143, Atlanta, Ga. to Nashville, Tenn.):

https://archive.org/stream/usairwaymap1931_143wash/uconn_magic_ocm658045082#page/n4/mode/thumb

you’ll see beacons numbered 14 and 15 in that general area. Does either of those look like it could be the one you’re referring to?

I also see a beacon on the USGS topo map of the Monteagle quad from 1936. There is a beacon very close to the county line. Check it out here: http://ims.er.usgs.gov/gda_services/download?item_id=5336184

**Reply by jengalas** [reply]
Posted by Patricia
January 24, 2014 at 4:05 pm
_Imported from WordPress_

Using the maps you sent me we have narrowed down the location. With the help from a friend who is a surveyor and his daughter who works in the geology dept. at the University of the South we are confident we can locate the spot where the tower stood. My husband remembered where a grouping of concrete pieces are located near the bluff. He said he always wondered why they were they. I raked back the leaves and found the concrete. It’s busted up but you can tell it hasn’t been piled there because it is laid out in a straight line. The pieces are at least 8 inches thick so I know they must have supported something heavy. It’s very exciting to know we are so close to finding out if this is the spot.
Thanks for your help.
Patricia

**Reply by jengalas** [reply]
Posted by Patricia
January 24, 2014 at 4:12 pm
_Imported from WordPress_

One more question. Do you know what they did with the towers when they were taken out of service? Did the owner come back and take them down or where they left for the landowner to use?
Thanks
Patricia

**Reply by jengalas** [reply]
Posted by Stormy
July 2, 2019 at 4:54 am
_Imported from WordPress_

How would I find the one in mobile?

**Reply by jengalas** [reply]
Posted by Don Gipson
February 15, 2021 at 4:30 pm
_Imported from WordPress_
 
Patricia,
My maternal grandparents lived in Bells Cove at the foot of Monteagle mountain. During WWll when visiting them as a youngster I would this very bright beacon in the North-Easterly direction at night. I always wondered where this beacon was located, After reading Zhanna’s reply and maps that beacon has to be # 15, near Pelham, TN. I do not know when that beacon was disabled. Sorry I am just now reading your quest.
Don

**Comment by jengalas** [comment]
Posted by Deskpilot
December 28, 2013 at 12:55 am
_Imported from WordPress_

I was turned onto this subject recently by the folks @ http://www.airtrafficatlanta.com/
So, in a matter of a few hours, I collected the data sets from this page and the WEST page and made a Google Fusion Table map of the beacon positions.
Here’s the link to the map. https://www.google.com/fusiontables/DataSource?docid=1dVWeFsTXp2eJzlSI13qKdgWv6G0jj2iMC4ibfAg

Each beacon point has all of the data associated with the NGS survey data.
Hop you enjoy browsing

**Reply by jengalas** [reply]
Posted by Eileen Stokes
April 14, 2014 at 5:26 am
_Imported from WordPress_

OMG–you made the map I have been looking for!! This helps soooo much. I have spent hours trying to piece together known beacons on this list and could tell there was more than one route involved.

I’d like to add one more beacon. It is long gone but not forgotten here in Washington Township, Morris County, NJ (Long Valley). However the exact spot was in danger of being lost so after years of collecting bits and pieces of local lore and getting the permission of neighbors in the area, we (on behalf of the Washington Township Historical Society) went out Saturday to pinpoint it–hoping the concrete pad (all that we were told remained) was still there under leaf mulch.
Success–the coordinates are: 40 44 32.2 (N) and 074 47 16.4 (W).

I can send a photo of the concrete pad, now in a wooded area on the Fox Hill Range, just off Beacon Hill Road in Long Valley, NJ. It is also marked on the 1943 topographic map that is on http://www.historicaerials.com/ and the spot can be seen on several of the aerial photos (1931, 1940 and 1953) although not recognizable as a tower.

The tower was decommissioned by WWII and used as a plan spotting tower by Civil Defense according to old timers. It was removed at some point after the war and the area became housing around 1958. According to the old timers it marked the route “to Chicago” and flashed red at regular intervals.

It lines up perfectly with the Beacon 66 listed on Montana Mountain (Belevidere, NJ) and then Beacon 65 in Kunklestown, PA and onto Cleveland and Chicago. I was led astray by information on the data sheet for Beacon 66 talking about the NY to Pittsburg route.

I have heard of former beacons in The Oranges and Morristown, NJ and they would also fall on that direct route. I have an inquiry in to Morris County Heritage Commission looking for information on the possible placement of those two beacons.

I love to get any more history on this route and its construction and dates. Thank you, thank you, thank you!!

**Reply by jengalas** [reply]
Posted by Barney
February 23, 2015 at 8:22 pm
_Imported from WordPress_

The generator shed, arrow and tower still exist at the Newark, Ohio airport – KVTA., The tower still has the course marker lights for the #2 beacon on the Columbus to Philadelphia route.

**Comment by jengalas** [comment]
Posted by Bill
December 17, 2013 at 12:03 pm
_Imported from WordPress_

Any suggestions on how to look for more info on the location of airway beacon #12 on the New York – Boston route? I have a 1940’s sectional chart that shows it just west of Willimantic, CT. I live nearby and have roughed out an approximate area in my town of where it might have been. Has anyone had any luck finding beacon location information outside of the NGS database?

Thanks.

**Comment by jengalas** [comment]
Posted by Robbie
December 2, 2013 at 8:28 pm
_Imported from WordPress_

Very interested in several beacon locations not listed here

Waterman IL (Where US Rt 30 and IL Rt 23 meet – site of the old Waterman airport emergency field)
Possibly one in Scarborough IL??
North of Big Rock IL (Possibly beacon #39)

I live oin Chicagoland and am interested in all the beacons west of the old North Aurora airport

Thanks!

**Comment by jengalas** [comment]
Posted by Bob Pastore
November 30, 2013 at 6:34 pm
_Imported from WordPress_

There was an Airway Beacon at Summit Hill, PA. I have a 1956 FAA Sectional Chart with the beacon shown. Do you want a copy?

**Reply by jengalas** [reply]
Posted by Zhanna
November 30, 2013 at 7:15 pm
_Imported from WordPress_

I would certainly like a copy! I will send you an e-mail. Thanks!

**Comment by jengalas** [comment]
Posted by Jack Place
November 21, 2013 at 7:47 pm
_Imported from WordPress_

BCN 17 on Avery mountain overlooking Tunkhannock PA has an elevation of 1879 Ft. I haven’t been there in 40 some years, not sure it’s still there. Cant see anything from Google Earth, too overgrown. I used to have an old discarded 1000w bulb from the beacon.

**Reply by jengalas** [reply]
Posted by Zhanna
November 21, 2013 at 9:07 pm
_Imported from WordPress_

This is one beacon site I’ve actually been to!

Details of the visit and photographs of the remains of the beacon are here: [LY2802](https://thesurveystation.com/surveymarks/ly2802).

Also at the site is triangulation station OSTER: [LY2803](https://thesurveystation.com/surveymarks/ly2803).

Any chance you have a photograph of the old bulb? That would make a nice addition to the page.

Thanks very much!

**Comment by jengalas** [comment]
Posted by hrdwrkr
November 21, 2013 at 12:31 pm
_Imported from WordPress_

ND0230 in Erie, PA was on the grounds of Erie International Airport.
The beacon was displaced by a communications building as seen on Google Maps.
http://goo.gl/maps/VLgQ2

But alas, you can add ne more beacon to the destroyed list.
The Bing Maps images show that all traces of the beacon and ground pad are now gone.
http://binged.it/1e41Y8j
Maybe someone can check if a benchmark or some other note was left behind.
Parts of the “entrance road” that is referenced in the NGS Data Sheet are still visible in both pictures.

**Comment by jengalas** [comment]
Posted by Doug
November 19, 2013 at 1:46 pm
_Imported from WordPress_

Hi.. Just found this site and live in Arizona… I plugged in 3 of the coords from the list above for sites in Arizona…. What I got in all 3 are roads.. Are the coords just approximate? Or am I finding sites that no longer exist? I would love to help find thses but not sure if I need to do a grid search using the coords you have… Maybe i need some help in understanding the info… Thanks

**Reply by jengalas** [reply]
Posted by Zhanna
November 19, 2013 at 3:42 pm
_Imported from WordPress_

The coordinates should be exact. Which ones were you looking for? It’s possible that roads were later built on the beacon sites once the beacons were removed. Also, be aware that Google Maps will typically show you two markers when you enter coordinates. The green arrow indicates the coordinates that you entered, and the red “A” bubble shows the nearest geocoded address. For the beacon sites, you want to look at the green arrow marker.

**Comment by jengalas** [comment]
Posted by Noell Wilson
November 19, 2013 at 12:28 pm
_Imported from WordPress_

I have seen the concrete remains of the Ft Worth ATL AWY BCN 75 between Carrollton and Villa Rica GA about 30 years ago. The concrete base is less than 30 feet off the west side of Shackelford Rd. about 1/4 mile north of West Hickory Level Rd zip 30116. There was a private airstrip owned by a friend of mine who was a flying instructor in WW II – Zellie Morgan. Zellie said that, before there were lights everywhere, you could get some altitude and see several of these lights in a row when flying cross country. At that time he was flying a Taylorcraft with no radio and following the roads and railroads. My eyeball on Google maps was within a few seconds of the official location. You can still make out Zellie’s grass airstrip extending to the west of the beacon location.

**Comment by jengalas** [comment]
Posted by Andy Mabbett
November 18, 2013 at 4:41 pm
_Imported from WordPress_

I’ve just added one of these arrows to OpenStreetMap – http://www.openstreetmap.org/browse/way/246901872 – and would encourage others to do likewise for the remaining examples. OpenStreetMap is a wiki – anyone can edit it.

**Comment by jengalas** [comment]
Posted by Wordsmyth8@aol.com
November 18, 2013 at 3:10 pm
_Imported from WordPress_

I am disappointed that the coordinates are not listed for the “cement arrow” that I’ve been told exists in the Michaux State Forest in southcentral Pennsylvania. A district forester told me about this arrow. From the way he explained it, it should be in Adams County, off Piney Mountain Ridge Road in Michaux. If anyone has any information at all about the arrow or has the GPS coordinates, I would love to know. Thanks.

**Reply by jengalas** [reply]
Posted by Zhanna
November 18, 2013 at 3:57 pm
_Imported from WordPress_

Hi there,

There may be more, but a quick Google search resulted in this page: [Take a day trip to the Caledonia State Park area](http://ydtalk.com/rocks/?p=284), which has some photographs of the arrow. GPS coordinates are given as N 39.906473° W 77.436702°.

**Reply by jengalas** [reply]
Posted by Herbie
January 8, 2016 at 8:14 pm
_Imported from WordPress_

still there 1-8-16 N39.90688 W77.43632 on my GPS Herbie out

**Comment by jengalas** [comment]
Posted by Lou
November 18, 2013 at 12:12 pm
_Imported from WordPress_

How do I find the latitude and longitude of a site?

**Comment by jengalas** [comment]
Posted by Drew T
November 18, 2013 at 12:44 am
_Imported from WordPress_

Beacon at HV4584 is gone but not forgotten. This was the location of Beacon Hill Airport, located roughly in between George Washington’s Mt. Vernon and Old Town Alexandria along historic Rt. 1 in Alexandria, VA. It appears that the beacon preceded the construction of the airport at this intersection in 1929, hence the name “Beacon” appearing in the airport’s name. Lots of history at http://www.beaconfieldairport.com including stories about the Groveton Observation Post located here during WWII. The airport was closed in 1959 and is a strip mall now. A historic designation marker is the only remnant of this beacon. Thank you for your intriguing site, Hanna!

**Comment by jengalas** [comment]
Posted by Chad
November 17, 2013 at 10:33 pm
_Imported from WordPress_

The nearest to me is JZ3091. When I added the Latitude and Longitude on Google Earth, it was a farm in Ohio. Does anyone happen to know if there’s still an arrow there?

**Comment by jengalas** [comment]
Posted by William Ryall
November 17, 2013 at 10:27 pm
_Imported from WordPress_

We have a weekend place at 101 High Ridge Road, Dingmans Ferry, Pa. Our property borders The Delaware Water Gap forest. Directly behind our home about 1/4 mile is an arrow and the concrete base of the light tower. I had no idea what this was until now. Very exciting.

**Reply by jengalas** [reply]
Posted by Wordsmyth8@aol.com
November 18, 2013 at 3:11 pm
_Imported from WordPress_

Cool!

**Reply by jengalas** [reply]
Posted by Zhanna
November 18, 2013 at 3:14 pm
_Imported from WordPress_

That’s definitely exciting! Do you have coordinates for the arrow? I can’t see much on the satellite view because the area is heavily wooded. The only beacon in Pike County on my NGS list (LY2602) is not the one near you.

**Comment by jengalas** [comment]
Posted by harry
November 17, 2013 at 10:08 am
_Imported from WordPress_

Are there any in the cookeville tn area?

**Reply by jengalas** [reply]
Posted by Zhanna
November 17, 2013 at 10:40 am
_Imported from WordPress_

There may be, but there are none listed in the NGS database in Putnam County. I can put you in touch with someone else in Tennessee who might be able to help you out!

**Reply by jengalas** [reply]
Posted by Betty
November 18, 2013 at 9:39 am
_Imported from WordPress_

Would love to know any contacts or further info you have for these arrows in TN, AL, or KY!! Thx!

**Reply by jengalas** [reply]
Posted by Zhanna
November 18, 2013 at 3:27 pm
_Imported from WordPress_

Hi Betty,

Dick Merrill has submitted two reports on beacons in TN and KY. They are posted on my site: [Airway Beacon 25 near Johnson City, TN](https://thesurveystation.netlify.app/2013/11/10/airway-beacon-25-near-johnson-city-tn) and [Kentucky Airway Beacon 70 in Warren County, KY](https://thesurveystation.netlify.app/2013/11/10/kentucky-airway-beacon-70-in-warren-county-ky). I can also send you his contact information. Thanks for visiting and commenting!

**Comment by jengalas** [comment]
Posted by Kat Stevens
November 16, 2013 at 10:24 am
_Imported from WordPress_

LX6975 in Cromwell, CT is someone’s back yard. Not a trace of anything remaining.

**Reply by jengalas** [reply]
Posted by rosemarie querns
November 17, 2013 at 11:25 pm
_Imported from WordPress_

Really? Do you think they know it was there? I live real close in Durham.

**Comment by jengalas** [comment]
Posted by Al Kamminga
August 3, 2013 at 11:30 am
_Imported from WordPress_

Looking for beacons 21 near Walcott, IN, 22A Pembroke, IN Newton County, Section 25, Township 31N, 22B near Shelby, IN, and one near Lowell, IN on your list..
Did they all have a 53 foot concrete arrow?
Beacon 22A was hooked up to REMC power August 8, 1939 and energized one month later. The electric load was estimated at 1650 watts continuous from dusk to dawn. The beacon was turned off in 1950.
Emery-Riddle Company (ERC) forerunner of American Airlines began flying the Chicago-Indianapolis-Cincinnati route daily on Dec 17, 1927.
My source is the Jasper County REMC Electric Consumer August 2013 Volume 63 number 2. http://www.ElectricConsumer.org.

**Comment by jengalas** [comment]
Posted by Gary Holden
February 15, 2013 at 10:49 am
_Imported from WordPress_

Hiked up to the site of MZ1975 on the top of Turner Mtn in Stephentown, NY in the spring of 2011. No sign of any tower but the concrete pad remains as does evidence of the electrical service to the site from the south west.

**Reply by jengalas** [reply]
Posted by Zhanna
February 15, 2013 at 5:36 pm
_Imported from WordPress_

Thanks for the update on MZ1975! Do you have any photos of the area that you could share? You might also want to consider submitting a report to NGS explaining your discovery; they would make the determination whether to update the database to mark the beacon Destroyed.
